### PR TITLE
style: add more comments before audit

### PR DESCRIPTION
**Pull request by 0xRusowsky**

### CODE REVIEW

File: src/Clearinghouse.sol [hunk 1]
```diff
@@ -78,17 +80,20 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
...
         address kernel_
     ) Policy(Kernel(kernel_)) CoolerCallback(coolerFactory_) {
+        // Store the relevant contracts.
         gOHM = ERC20(gohm_);
         staking = IStaking(staking_);
         sdai = ERC4626(sdai_);
         dai = ERC20(sdai.asset());
         
-        // Initialize the contract status and its funding schedule.
+        // Initialize the contract status and its funging schedule.
```
Comment (ind-igo) [inline src/Clearinghouse.sol]: Typo, should be `funding`

File: src/Clearinghouse.sol [hunk 2]
```diff
@@ -329,6 +336,7 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
     /// @notice Return funds to treasury.
     /// @param  token_ to transfer.
     /// @param  amount_ to transfer.
+    /// @dev    Access control is storage is held in the `ROLES` module.
```
Comment (ind-igo) [inline src/Clearinghouse.sol:339]: Unnecessary imo.

File: src/Clearinghouse.sol [hunk 3]
```diff
@@ -350,6 +358,7 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
     }
 
     /// @notice Deactivate the contract and return funds to treasury.
+    /// @dev    Access control is storage is held in the `ROLES` module.
```
Comment (ind-igo) [inline src/Clearinghouse.sol:361]: Same as above

File: src/Clearinghouse.sol [hunk 4]
```diff
@@ -365,6 +374,7 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
     }
 
     /// @notice Reactivate the contract.
+    /// @dev    Access control is storage is held in the `ROLES` module.
```
Comment (ind-igo) [inline src/Clearinghouse.sol:377]: Same as above.

File: src/Cooler.sol [hunk 5]
```diff
@@ -220,17 +235,19 @@ contract Cooler is Clone {
...
+        // IF necessary, ensure lender implements the CoolerCallback abstract.
         if (isCallback_ && !CoolerCallback(msg.sender).isCoolerCallback()) revert NotCoolerCallback();
 
         // Ensure loan request is active. 
         if (!req.active) revert Deactivated();
 
-        // Clear the loan request.
+        // Clear the loan request in both, memory and storage.
         req.active = false;
+        requests[reqID_].active = false;
```
Comment (ind-igo) [inline src/Cooler.sol:248]: Can do this at the end of the function for clarity and keeping with CEI pattern

**Review by ind-igo — CHANGES_REQUESTED** [review]

File: src/Clearinghouse.sol (hunk 1 above)
Comment (0xRusowsky) [reply src/Clearinghouse.sol]: done

**Pull request merged**